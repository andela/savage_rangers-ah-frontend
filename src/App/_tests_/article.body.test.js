import React from 'react';
import { shallow } from '../../enzyme';
import ArticleBody from '../Components/DisplayArticle/articleBody';

const data = {
  title: 'software',
  body: 'Software',
  readTime: '2min',
  createdAt: '2019-08-14T12:56:55.150Z',
  coverImage: '',
  firstName: 'Frank',
  lastName: 'Mutabazi',
  profileImage: ''
};

describe('testing the article body component', () => {
  const articleBody = shallow(<ArticleBody
    title={data.title}
    body={data.body}
    readTime={data.readTime}
    createdAt={data.createdAt}
    coverImage={data.coverImage}
    firstName={data.firstName}
    lastName={data.lastName}
    profileImage={data.profileImage}
  />);
  it('should test if the component renders the appropriate jsx elements', () => {
    expect(articleBody.find('div').exists()).toBe(true);
    expect(articleBody).toMatchSnapshot();
  });
});
