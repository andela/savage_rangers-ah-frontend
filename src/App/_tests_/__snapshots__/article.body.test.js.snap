// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the article body component should test if the component renders the appropriate jsx elements 1`] = `ShallowWrapper {}`;
